docs/sources/whatsnew/whats-new-in-v7-3.md

```diff
@@ -59,6 +60,11 @@ As this new option is a standard field option it works in every panel. Here is a
 
 {{< figure src="/img/docs/v73/bar_gauge_gradient_color_scheme.png" max-width="900px" caption="bar gauge color scheme" >}}
 
+## CSV exports for Excel
+
+In v7.0, we introduced a new table panel and inspect mode with Download CSV enabled. However, CSV export to Excel was removed. As it turns out, Grafana users love their Excel! After many requests, this [community contribution from tomdaly](https://github.com/grafana/grafana/pull/27284) brought it back.
```

Comment:
Instead of "As it turns out, Grafana users love their Excel! After many requests, this [community contribution from tomdaly](https://github.com/grafana/grafana/pull/27284) brought it back.", how about:

Due to a large number of inquiries and requests, this [community contribution from tomdaly](https://github.com/grafana/grafana/pull/27284) brought the feature back.

Reply:
I like this!